src/test/java/org/sagebionetworks/web/unitclient/widget/table/v2/schema/ColumnModelTableRowEditorWidgetTest.java

```diff
@@ -78,6 +83,7 @@ public void testConfigure() {
 	public void testTypeChange() {
 		editor.configure(columnModel, null);
 		reset(mockView);
+		when(mockView.getMaxSize()).thenReturn("");

```

Comment:
Since the mockView is fully reset, I need to re-initialize the default behavior when asking for the max size (in reality this is a text element in the dom, that will always return the empty string unless it contains a different value).